cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -833,6 +835,10 @@ public Retry.ActionFailedException convertToActionFailedExceptionCausedByCloudCo
         return Retry.ActionFailedException.ofCause(convertToCloudConnectorException(e, actionDescription));
     }
 
+    public boolean isExceptionCodeConflict(ManagementException e) {
```

Comment:
There is a class [AzureExceptionHandler](https://github.com/hortonworks/cloudbreak/blob/42ca805420308741856a4474840a1396a1a8c807/cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/util/AzureExceptionHandler.java#L78-L93) where there are utility methods to check an exception's status code: